stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -581,7 +581,11 @@ end
     finally
         ccall((:CloseClipboard, "user32"), stdcall, Cint, ()) == 0 && Base.windowserror("CloseClipboard")
     end
-    @test clipboard() == ""
+    if get(ENV, "BUILDKITE", "") == "true"
+        @test_broken clipboard() == "" # TODO: fix this test on Buildkite CI
+    else
+        @test clipboard() == ""
+    end
```

Comment:
This isn't going to be merged as is as per comment above, but for the record this can be simplified to
```julia
@test clipboard() == "" broken=get(ENV, "BUILDKITE", "")=="true"
```